code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -86,9 +86,9 @@ \subsection{Table of Symbols}
 \\
 ${{F_{S}}^{min}}$ & Minimum Factor of Safety: The minimum factor of safety associated with the critical slip surface & --
 \\
-${F_{x}}$ & X-Component of the Force:  & N
+${F_{x}}$ & $x$-component Of the Force:  & N
 \\
-${F_{y}}$ & Y-Component of the Force:  & N
+${F_{y}}$ & $y$-component Of the Force:  & N
```

Comment:
The capitalization of "Of" in these rows is because the `NounPhrase` has to be built from a `Sentence` to have the *x* and *y*. I think in the future the SSP ToS should be revisited, since it is the only example that has the name of the quantity followed by its definition, so this could be left for now? @smiths

Reply:
EDIT: There is a lot of in-depth instances where `xDir`, etc. should be used in SSP, so I won't do that in this PR.

Reply:
When you ask whether this should be left for now, what do you mean?  The capitalization of "Of"?  It does look odd to have a capital like that.  We should create an issue related to this so that we don't forget it is something that we want to eventually fix.

Reply:
Yes, that is what I meant, but at least for now (in the Projectile branch), I switched SSP back from using `yComp`, etc. to avoid this for now. I'll make an issue for swapping SSP over to using these concepts.

Reply:
Sounds good.  Thank you @samm82.

Reply:
Issue made at #1626